scrapers.js

```diff
@@ -1686,6 +1686,45 @@ let scrapers = [
       };
     }
   },
+  {
+    country: 'FRA',
+    url: 'https://raw.githubusercontent.com/opencovid19-fr/data/master/dist/chiffres-cles.csv',
+    priority: 1,
```

Comment:
Note: Add `timeseries: true` when sources have timeseries data, helps them win in conflicts.